zerver/actions/streams.py

```diff
@@ -72,44 +72,6 @@
 from zerver.tornado.django_api import send_event, send_event_on_commit
 
 
-def send_user_remove_events_on_stream_deactivation(
```

Comment:
are we updating the message displayed when we're archiving a stream ?
if so, lets add it in this commit.
```
Archiving channel #channelname will immediately unsubscribe everyone. This action cannot be undone.
```

Reply:
I asked [CZO](https://chat.zulip.org/#narrow/stream/101-design/topic/archived.20streams.20in.20Streams.20menu/near/1849896) for what the updated message should be. For now i have added the changes related updating the message in this commit.